### ISSUE TITLE
Check Internet Provider for new office

### ISSUE THREAD

**Issue by writerity-watanabe** [timeriver/company-management#307]
i think we need another one and stronger one

**Comment by writerity-t-bojan** [comment]
@writerity-watanabe I made a spreadsheet with two offers for now. About the last offer, I contacted them again and I'll make an update as soon as I have information.
https://docs.google.com/spreadsheets/d/18-K6ZtdB1pSTt3zr5Douq23nbVOZyjT-XPqMjA6LdrE/edit#gid=0

**Comment by writerity-watanabe** [comment]
@writerity-t-bojan 
thank you.
can you check T-mobile as well?

IP address should be not dynamic btw if possible

**Comment by writerity-t-bojan** [comment]
@writerity-watanabe I talked with Neotel about the dynamic IP address and they told me there is no possibility to change that. Fiber 100 and Fiber 300 are packages for private users and have this kind of address. Office Fiber 80 and Office Fiber 100 are for companies and these are without dynamic IP address. I updated the sheet with another offer as well.
 As far as T-mobile goes I contacted them and I'm waiting an offer.
